src/routes/productRoutes.ts

```diff
@@ -8,6 +8,11 @@ import isAuthenticated from '../middleware/isAuthMiddleware';
 import checkPermission from '../middleware/checkPermissionMiddleware';
 import upload from '../middleware/multerConfig';
 import { productController } from '../controllers/productStatusController';
+import ProductController from '../controllers/updtproductController';
+import multer from 'multer';
+import authMiddleware from '../middleware/authMiddleware';
+import uploads from '../middleware/multer';
+import cloudinary from '../helps/cloudinaryConfig';
 
 const router = express.Router();
 router.get('/', getProducts);
```

Comment:
Admins should be the ones to see products using this `getProducts` function since it gets all products regardless of the expiration or the availability of the product.